src/CoreWCF.Primitives/src/CoreWCF/ServiceSecurityContext.cs

```diff
@@ -159,6 +160,12 @@ public AuthorizationContext AuthorizationContext
             }
         }
 
+        public ClaimsPrincipal ClaimsPrincipal => _authorizationContext != null
+                   && _authorizationContext.Properties.TryGetValue("ClaimsPrincipal", out object principal)
```

Comment:
We already have a ClaimsPrincipal property on OperationContext. We aren't currently setting it as the code in `DispatchOperationRuntime.SetClaimsPrincipalToOperationContext` hasn't been re-enabled yet.

Reply:
done in https://github.com/CoreWCF/CoreWCF/pull/850/commits/0da8aca434f3bcb07d3fd55955a69d39cb3cd352